pilot/pkg/model/context.go

```diff
@@ -278,6 +278,8 @@ type NodeMetadata struct {
 	TLSClientKey string `json:"TLS_CLIENT_KEY,omitempty"`
 	// TLSClientRootCert is the absolute path to client root cert file
 	TLSClientRootCert string `json:"TLS_CLIENT_ROOT_CERT,omitempty"`
+	// DNSCert is the absolute path to a directory containing an additional non citadel issued X509 certificate with DNS SANs
```

Comment:
This is a server cert right? Can we call this `TLSServerDNSCert`. **TLS** and **Server** to maintain consistency.